lib/ansible/modules/cloud/vmware/vmware_host.py

```diff
@@ -50,11 +50,12 @@
     - "For example, if there is a datacenter 'dc1' under folder called 'Site1' then, this value will be '/Site1/dc1/host'."
     - "Here 'host' is an invisible folder under VMware Web Client."
     - "Another example, if there is a nested folder structure like '/myhosts/india/pune' under
-       datacenter 'dc2', then C(folder) value will be '/dc2/host/myhosts/india/pune'."
+       datacenter 'dc2', then C(folder_name) value will be '/dc2/host/myhosts/india/pune'."
     - "Other Examples: "
     - "  - '/Site2/dc2/Asia-Cluster/host'"
     - "  - '/dc3/Asia-Cluster/host'"
     version_added: "2.6"
+    aliases: ['folder_name']
```

Comment:
I am ok to change "folder" to "folder_name" as the option for the datacenter name is "datacenter_name".
But if you do, do it everywhere (option name and python variable) and add an alias "folder" for retro-compatibility.

Reply:
I would leave it like this. I don't see what `_name` adds except maybe confusion. Obviously it's the folder you have to provide. I would leave it at `C(folder)` in the description.

Reply:
that's also ok for me to leave it like thisii

Reply:
The "problem" with all the vmware modules is that they are not consistent. One module uses e.g. datacenter_name and the other uses only datacenter and so on.

Reply:
I think we need to have consistency for all the VMware-related modules wrt. this. And my proposal is to not use `_name` for the standard parameters. (We can have them as aliases though)

Reply:
I will put it on the agenda for next week's meeting.

Reply:
Ok. I had the impression that parameters with _name are more used (datacenter _name, cluster_name, switch_name, etc. pp. ). I'm fine if we can achieve consistency.So, for example, you recommend to change datacenter_name to datacenter and (only) add an alias for backward compatibility?

Reply:
Changed back to folder, but leave alias.